base_layer/wallet/migrations/2021-03-23-082938_update-outputs-for-tari-script/up.sql

```diff
@@ -1,12 +1,12 @@
 -- This migration is part of a testnet reset and should not be used on db's with existing old data in them
 -- thus this migration does not accommodate db's with existing rows.
 ALTER TABLE outputs
-    ADD COLUMN script BLOB NOT NULL;
+    ADD COLUMN script BLOB NULL DEFAULT NULL;
```

Comment:
NULL columns are NULL by default, the `DEFAULT NULL` adds a constraint that does nothing